src/Api/Utilities/DeviceUpdateHandler.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Bit.Core.Services;
+
+namespace Bit.Api.Utilities;
+
+
+public class DeviceUpdateHandler
+{
+
+    public IDeviceService _deviceService;
+    public IPushRegistrationService _pushRegistrationService;
+
+}
```

Comment:
Where is this used? I can't find the reference to it

Reply:
Oops.  I deleted the file from the solution but not the hard drive.  I'll clean it up.